pkg/storage/storage.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package storage // import "k8s.io/helm/pkg/storage"
+
+import (
+	rspb "k8s.io/helm/pkg/proto/hapi/release"
+	"k8s.io/helm/pkg/storage/driver"
+)
+
+type Storage struct {
+	driver.Driver
+}
+
+// Create creates a new storage entry holding the release.
+// An error is returned if storage failed to store the release.
+//
+// TODO: Is marking the release as deployed the only task here?
+// 		 What happens if an identical release already exists?
+func (s *Storage) Create(rls *rspb.Release) error {
```

Comment:
Usually, a database driver will fail if Create is called with an existing key. So we should probably do that.